recipe/meta.yaml

```diff
@@ -2,7 +2,7 @@
 {% set version = "1.13.1" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-suite
```

Comment:
Oh man. I like `suite` so much more than `split`. Thank you for giving me another name to use here for my multiple output recipes!

Reply:
`-suite` is what the mpl recipe uses.